src/Symfony/Component/Form/Form.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Symfony\Component\Form;
 
-use Symfony\Component\Form\Exception\RuntimeException;
```

Comment:
please don't do those changes in 2.8 as it can cause conflicts when merging later